ayon_bin_distro/lakectlpy/wrapper.py

```diff
@@ -93,6 +126,14 @@ def help(self):
             sys.stdout.write(process.stdout.readline())
 
     def list_repo_objects(self, lake_fs_repo_uri: str):
+        """list objects on a given repository
+
+        Args:
+            lake_fs_repo_uri: lakeFs internal uir for the repository
```

Comment:
```suggestion
            lake_fs_repo_uri: lakeFs internal uri for the repository
```
Typo. Also, what makes it "internal"?